tests/framework/e2e/util.go

```diff
@@ -50,7 +50,10 @@ func SpawnWithExpectWithEnv(args []string, envVars map[string]string, expected s
 }
 
 func SpawnWithExpects(args []string, envVars map[string]string, xs ...string) error {
-	_, err := SpawnWithExpectLines(context.TODO(), args, envVars, xs...)
+	ctx, cancel := context.WithTimeout(context.TODO(), 2*time.Second)
```

Comment:
There is no expectation that this commands should only take 2 seconds. I expect because this function is mostly used for running `etcdctl` it work, but the the name of the function do not suggest limited run time. 

If limiting run time is required for tests to pass, please move it up the call stack.

Reply:
removed again